error: non-constant-expression cannot be narrowed from type 'uint64_t' (aka 'unsigned long long') to 'size_type' (aka 'unsigned int') in initializer list [-Wc++11-narrowing]

Building is broken on 32-bits (FreeBSD 14.0-RELEASE / powerpc, armv7, i386):
```
/wrkdirs/usr/ports/databases/tiledb/work/TileDB-2.19.0/tiledb/../tiledb/sm/array_schema/enumeration.h:309:50: error: non-constant-expression cannot be narrowed from type 'uint64_t' (aka 'unsigned long long') to 'size_type' (aka 'unsigned int') in initializer list [-Wc++11-narrowing]
    return {static_cast<uint8_t*>(data_.data()), data_.size()};
                                                 ^~~~~~~~~~~~
/wrkdirs/usr/ports/databases/tiledb/work/TileDB-2.19.0/tiledb/../tiledb/sm/array_schema/enumeration.h:309:50: note: insert an explicit cast to silence this issue
    return {static_cast<uint8_t*>(data_.data()), data_.size()};
                                                 ^~~~~~~~~~~~
                                                 static_cast<size_type>( )
/wrkdirs/usr/ports/databases/tiledb/work/TileDB-2.19.0/tiledb/../tiledb/sm/array_schema/enumeration.h:319:53: error: non-constant-expression cannot be narrowed from type 'uint64_t' (aka 'unsigned long long') to 'size_type' (aka 'unsigned int') in initializer list [-Wc++11-narrowing]
    return {static_cast<uint8_t*>(offsets_.data()), offsets_.size()};
                                                    ^~~~~~~~~~~~~~~
/wrkdirs/usr/ports/databases/tiledb/work/TileDB-2.19.0/tiledb/../tiledb/sm/array_schema/enumeration.h:319:53: note: insert an explicit cast to silence this issue
    return {static_cast<uint8_t*>(offsets_.data()), offsets_.size()};
                                                    ^~~~~~~~~~~~~~~
                                                    static_cast<size_type>( )
2 errors generated.
```

--- a/tiledb/sm/array_schema/enumeration.h
+++ b/tiledb/sm/array_schema/enumeration.h
@@ -306,7 +306,8 @@ class Enumeration {
              size of the buffer pointed to.
    */
   span<uint8_t> data() const {
-    return {static_cast<uint8_t*>(data_.data()), data_.size()};
+    return {
+        static_cast<uint8_t*>(data_.data()), static_cast<size_t>(data_.size())};
   }
 
   /**
@@ -316,7 +317,9 @@ class Enumeration {
    *         the size of the buffer pointed to.
    */
   span<uint8_t> offsets() const {
-    return {static_cast<uint8_t*>(offsets_.data()), offsets_.size()};
+    return {
+        static_cast<uint8_t*>(offsets_.data()),
+        static_cast<size_t>(offsets_.size())};
   }
 
   /**

